client/src/pages/Profiles/Profiles.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, TextField, Typography, Box } from '@mui/material';
+import useStyles from './useStyles';
+import hero from '../../images/landing/hero.jpg';
+import { useEffect, useState } from 'react';
+import { getProfiles } from '../../helpers/APICalls/getProfiles';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import ProfileCard from './ProfileCard/ProfileCard';
+
+export default function Profiles() {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [profiles, setProfiles] = useState([]);
+
+  useEffect(() => {
+    const loadProfiles = async () => {
+      const profiles = await getProfiles();
```

Comment:
Since we're performing an async operation, let's wrap this in a try/catch block to handle promise rejections